study.md

```diff
@@ -30,32 +30,44 @@ Please note:
 In your own words, define what the responsibilities of the model layer are in
 Rails.
 
-```md
-<!-- your answer here -->
-```
+
+The model layer in Rails are Ruby classes. It communicates with the database, as well as store and validates data. It handles domain concepts and business logic. It works with the controller to deliver what the user HTTP-requests.
+
+-   [Purpose of Rails Controller](http://techiferous.com/2013/04/so-what-exactly-is-the-purpose-of-a-rails-controller/)
+-   [Ruby On Rails Getting Started](https://en.wikibooks.org/wiki/Ruby_on_Rails/Getting_Started/Model-View-Controller#Model)
 
 ## Define Controller Responsiblities
 
 In your own words, define what the responsibilities of the controller layer are
 in Rails.
 
-```md
-<!-- your answer here -->
-```
+
+When users visit a Rails web application, users are routed to one of many controllers. Controllers connects Models and Views. They respond to user HTTP requests, translate what's in the model/database to the server that then displays a *view*able format for the user.
```

Comment:
Requests are routed to controllers, yes, but more precisely they are routed to actions within a controller, called a "controller action" or sometimes just "action".